src/os/ui/featureedit.js

```diff
@@ -739,6 +744,25 @@ os.ui.FeatureEditCtrl.prototype.isEllipse = function() {
 };
 
 
+/**
+ * If a polygon is selected.
+ *
+ * @return {boolean}
+ * @export
+ */
+os.ui.FeatureEditCtrl.prototype.isPolygon = function() {
+  var geometry = this.previewFeature.getGeometry();
+
+  if (geometry) {
+    var type = geometry.getType();
```

Comment:
This duplicates `os.geo.isGeometryPolygonal`. You can use that instead.

Reply:
This has been updated.